webapp/channels/src/components/header_footer_template/header_footer_template.tsx

```diff
@@ -108,10 +99,7 @@ export default class NotLoggedIn extends React.PureComponent<Props> {
                     location='header_footer_template'
                     href={this.props.config.HelpLink}
                 >
-                    <FormattedMessage
-                        id='web.footer.help'
-                        defaultMessage='Help'
-                    />
+                    {localizeMessage('web.footer.help', 'Help')}
```

Comment:
This is to remove unnecessary `span` element. No side-effect in UI but made for ease of testing.

Side note: This is being used in reset_password page like below.
<img width="742" alt="Screenshot 2023-08-08 at 6 52 49 AM" src="https://github.com/mattermost/mattermost/assets/5334504/83585301-2cd4-4b52-b758-26f71beb2409">

but maybe make sense to follow the new footer like in login or signup pages.
<img width="769" alt="Screenshot 2023-08-08 at 6 52 38 AM" src="https://github.com/mattermost/mattermost/assets/5334504/1b627d07-37a2-4352-8324-05286f0c3cb8">

Reply:
would it be possible to use intl props instead of this util functions. We are trying to get rid of them eventually.

Reply:
Can you use formatMessage from injectIntl here, we are trying to slowly reduce the usage of custom i18n functions. I will make a post about it soon in channel

Reply:
PR submitted - https://github.com/mattermost/mattermost/pull/24244